dothething/dothething/ClipsView.swift

```diff
@@ -35,71 +34,101 @@ struct ClipsView: View {
     @Binding var toggle: Bool
     
     var body: some View {
-        VStack {
-            HStack {
-                // back button
-                Button(action: {
-                    print("Back button tapped")
-                    clipsViewModel.backButtonPressed()
-                    toggle.toggle()
-                }) {
-                    Text("Back")
-                }                                                                                                                                                                                                                                                                                                                                                                                                                                   
-                .padding(.leading)
-                .frame(maxWidth: .infinity, alignment: .leading)
+        GeometryReader { geometry in
+            VStack {
+                ZStack {
+                    HStack {
+                        Button(action: {
+                            print("Back button tapped")
+                            clipsViewModel.backButtonPressed()
+                            toggle.toggle()
+                        }) {
+                            Image(systemName: "chevron.left")
+                                .font(.system(size: 24))
+                                .foregroundColor(.accentColor)
+                        }
+                        .padding(.leading)
+                        Spacer()
+                    }
+                    
+                    Text("domino")
+                        .font(.custom("Montserrat-Medium", size: 27))
+                        .foregroundColor(Color.accentColor)
+                        .tracking(8)
+                        .multilineTextAlignment(.center)
+                }
```

Comment:
The new code for the back button is an improvement, as it is now part of a ZStack with the "domino" title, which keeps the button in a consistent location relative to the content. The use of the SF Symbol for the back button icon is also a nice touch, making the UI more visually appealing.